src/index.ts

```diff
@@ -45,5 +45,5 @@ const executeCommand = async (): Promise<void> => {
 }
 
 executeCommand().catch((e: Error) => {
-  console.log('Error:', e.message)
-}).finally(() => process.exit())
+  console.error('Error:', e.message)
+}).finally(() => process.exit(1))
```

Comment:
Why exit with 1?
https://stackoverflow.com/questions/43147330/what-is-difference-between-method-process-exit1-and-process-exit0-in-node-js

Reply:
It's explained in the issue https://github.com/vacuumlabs/cardano-hw-cli/issues/23

Reply:
I see, but still, shouldnt it do `.then(() => process.exit(0)).catch((e) => process.exit(1))`

Reply:
Yes, I made mistake, fixed.